src/main/docs/guide/quickStart/creatingClient.adoc

```diff
@@ -2,7 +2,7 @@ As mentioned previously, Micronaut includes both an <<httpServer,HTTP server>> a
 
 .Testing Hello World
 
-snippet::io.micronaut.docs.server.intro.HelloControllerSpec[tags="imports,classinit,embeddedServer,class"]
+snippet::io.micronaut.docs.server.intro.HelloControllerSpec[tags="imports,classinit,embeddedServer,class",indent=0]
```

Comment:
Also adding this indent on creatingClient.adoc led to this PR https://github.com/micronaut-projects/micronaut-docs/pull/7 in micronaut-docs. So ideally if PR 7 & 8 can go together that would make these pages look nicer overall since the beans.adoc also uses that indent property.